README.md

```diff
@@ -117,19 +103,102 @@ The default newsletter template is:
 
 Take a look at existing articles for the various classes and where they're used.
 
-## Credits:
-
-- <a href="https://github.com/PartMan7" target="_blank">Parth Mane</a> (Lead)<br />
-- <a href="https://github.com/Goose-Of-War" target="_blank">Vidunram A R </a> (Contributor)<br />
-- <a href="https://github.com/nishkalprakash" target="_blank">Nishkal Prakash</a> (Contributor) <br />
-- <a href="https://github.com/MokshithPV" target="_blank">Venkatsai Mokshith</a> (Contributor)<br />
-- <a href="https://github.com/SachdevJai" target="_blank">Jai Sachdev</a> (Contributor)<br />
-- <a href="https://github.com/ItsAnkan" target="_blank">Ankan Saha</a> (Contributor)<br />
-- <a href="https://github.com/Symbiot01" target="_blank">Sahil Patel</a> (Contributor)<br />
-- <a href="https://github.com/KarmaAkaB" target="_blank">Soumil Maiti</a> (Contributor)<br />
-- <a href="https://github.com/anjaniit23" target="_blank">Anjani Kumar</a> (Contributor)<br />
-- <a href="https://github.com/lurkingryuu" target="_blank">Karthikeya Y M</a> (Contributor)<br />
-- <a href="https://github.com/Yureien" target="_blank">Soham Sen</a> (Contributor)<br />
-- <a href="https://github.com/ayush4ise" target="_blank">Ayush Parmar</a> (Contributor)<br />
-- <a href="https://github.com/Pagol1" target="_blank">Saumyadip Nandy</a> (Contributor)<br />
-- <a href="https://github.com/shiroyasha263" target="_blank">Vishesh Gupta</a> (Contributor)<br />
+---
+
+## Routers
+<!-- Note. This will be subject to change once the fs readdir router system is implemented. -->
+
+The Express app's router is set up by `/src/route.js`. The file route.js imports routers from the `/routes` folder. Each router file has the following format:
+
+```js
+const router = require('express').Router();
+// This router is used to configure the app for a specific route
+
+// GET requests
+app.get('/path', (req, res, next) => { 
+	// some code
+	return res.renderFile(template, ctx);
+});
+
+// POST requests
+app.post('/path', (req, res, next) => { 
+	// some code
+	return res.status(statusCode).send;
+});
+
+/* 
+Notes:
+	The next argument is optional in most cases. 
+	The functions in the requests can be asynchronous too.
+	You can use other routers as well if needed.
+*/
+
+module.exports = {
+	route: '/path'
+	router;
+}
+```
+
+All routers are imported and used in `route.js` in the `src` folder. 
+
+```js
```

Comment:
why 1 tab more in each line, this looks shifted by one tab tbh

Reply:
Because the actual code was ;-;
Should I shift it back?

Reply:
ig you should, cuz you are displaying just the code where the "thing" is happening

Reply:
nice